usr/share/rear/layout/save/GNU/Linux/230_filesystem_layout.sh

```diff
@@ -79,6 +79,14 @@ read_filesystems_command="$read_filesystems_command | sort -t ' ' -k 1,1 -u"
             Log "$device is CD/DVD type device [fstype=$fstype], skipping."
             continue
         fi
+        # docker specific exclude part
+        if service docker status >/dev/null 2>&1  ; then
```

Comment:
Currently
<pre>
if service docker status >/dev/null 2>&1 ; then
</pre>
is run again and again for each mounted filesystem because it is in the
<pre>
while read device mountpoint fstype options junk ; do
    ...
done < <( eval $read_filesystems_command )
</pre>
while loop.
I think it is better to run the if condition only once before
that while loop and store the condition result in a variable like
<pre>
local docker_is_running=""
service docker status >/dev/null 2>&1 && docker_is_running="yes"
...
while read device mountpoint fstype options junk ; do
    ...
    if is_true "$docker_is_running" ; then
        ...
    fi
    ...
done < <( eval $read_filesystems_command )
</pre>